ci/build-gdk.ps1

```diff
@@ -1,16 +1,21 @@
+# Expects gdk_home, which is not the GDK location in the engine
 param(
-  [string] $target_platform = "Win64"
+  [string] $unreal_path = "$((Get-Item `"$($PSScriptRoot)`").parent.parent.FullName)\UnrealEngine",
+  [string] $target_platform = "Win64",
+  [string] $build_output_dir
 )
 
 pushd "$($gdk_home)"
-
     Start-Event "build-unreal-gdk-$($target_platform)" "build-gdk"
     pushd "SpatialGDK"
-        $gdk_build_proc = Start-Process -PassThru -NoNewWindow -FilePath "$($gdk_home)\UnrealEngine\Engine\Build\BatchFiles\RunUAT.bat" -ArgumentList @(`
+        $gdk_build_proc = Start-Process -PassThru -NoNewWindow -FilePath "$unreal_path\Engine\Build\BatchFiles\RunUAT.bat" -ArgumentList @(`
             "BuildPlugin", `
-            " -Plugin=`"$($gdk_home)/SpatialGDK/SpatialGDK.uplugin`"", `
+            "-Plugin=`"$($gdk_home)/SpatialGDK/SpatialGDK.uplugin`"", `
             "-TargetPlatforms=$($target_platform)", `
-            "-Package=`"$($gdk_home)/SpatialGDK/Intermediate/BuildPackage/Win64`"" `
+                # Since we symlink the gdk into the engine, we have to put the build output outside of the GDK, 
+                # otherwise Unreal will find two instances of the plugin during the build process (It copies the .uplugin to the target folder at the start of the build process)
+                # Additionally, I ad some cases where BuildPlugin will fail when targeting a folder within UnrealEngine for output
```

Comment:
I think this sentence could be reworded, it sounds confusing to me.

Reply:
I have updated the wording here, but I am a terrible communicator.